specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/OnboardingStates.json

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
looks fine to me, but requires signoff from breaking change team.

Reply:
Thanks @zcxcasd 
Do we need to tag anyone to have a look?